staging/src/k8s.io/dynamic-resource-allocation/kubeletplugin/draplugin.go

```diff
@@ -73,6 +76,12 @@ type DRAPlugin interface {
 	internal()
 }
 
+type DRADriverService interface {
+	CheckDeviceAllocation(ctx context.Context, claims []*resourceapi.ResourceClaim) error
+	NodePrepareResources(context.Context, *resourceapi.ResourceClaim) ([]*draapi.AllocatedDevice, error)
+	NodeUnprepareResources(context.Context, *resourceapi.ResourceClaim) error
```

Comment:
This makes it look like the driver really gets a full ResourceClaim. That's not the case. Better pass what the caller has (UID, namespace, name) and document why the driver has to work without any additional information.